src/main/java/io/anserini/index/IndexCollection.java

```diff
@@ -638,6 +612,15 @@ public void destroyObject(PooledObject<RestHighLevelClient> pooled) throws Excep
   private ObjectPool<SolrClient> solrPool;
   private ObjectPool<RestHighLevelClient> esPool;
 
+  private int getFileNumber(String fileName) {
```

Comment:
hrm... maybe move this into part of C4Collection somewhere?

Doesn't really belong here...

Reply:
moved into `DocumentCollection` class